lib/ansible/modules/system/authorized_key.py

```diff
@@ -152,6 +152,17 @@
     user: ubuntu
     state: present
     key: "{{ lookup('file', lookup('env','HOME') + '/.ssh/id_rsa.pub') }}"
+
+# Set up authorized_keys exclusively with many keys
```

Comment:
Please use multiline yaml and set a  `- name:` rather than a comment which is more readable and matches the other examples in this file